src/test/resources/standalone/config/router-config.json

```diff
@@ -237,8 +237,10 @@
       "sourceType": "bikely",
       "url": "https://api.safebikely.com/api/v1/s/locations",
       "headers": {
-        "X-Bikely-Token": "${BIKELY_TOKEN}",
-        "Authorization": "${BIKELY_AUTHORIZATION}"
+        // The environment variables do not exist on every computer using this config running tests
```

Comment:
In the `ExampleConfigTest` we use the following code so that you can add any kind of env variables into the examples without the tests failing: https://github.com/opentripplanner/OpenTripPlanner/blob/f38676a7d32117f7e604683f7d05f44dc0017f01/src/test/java/org/opentripplanner/standalone/config/ExampleConfigTest.java#L64-L68

Could we do something else here?

Reply:
It is not so easy, but I guess the `${BIKELY_TOKEN}` is just to indicate that the user must insert something there. There is no environment variable substitution going on in the old code so we can change it to "[BIKELY_TOKEN]".

Reply:
I changed it to `[BIKELY_TOKEN]` instead. In he doc it should be in code sections so it does not interfere with markdown links.